Sources/MapboxMaps/Gestures/GestureHandlers/PinchGestureHandler.swift

```diff
@@ -1,63 +1,70 @@
 import UIKit
 
-/// The PinchGestureHandler is responsible for all `pinch` related infrastructure
-/// Tells the view to update itself when required
-internal class PinchGestureHandler: GestureHandler {
-    private var previousScale: CGFloat = 0.0
+/// `PinchGestureHandler` updates the map camera in response to a 2-touch
+/// gesture that may consist of translation and scaling
+internal final class PinchGestureHandler: GestureHandler {
```

Comment:
Why is this gesture a final class?

Reply:
Because it is not designed to be subclassed.

Reply:
why is this the only gesture designed that way?

Reply:
Good catch, they should all be final except for the base class.

Reply:
Updated